README.md

```diff
@@ -12,7 +12,7 @@ PHP client library for [Coveralls](https://coveralls.io).
 
 # Prerequisites
 
-- PHP 5.3 or later
+- PHP 5.5+ for 2.x or 5.2+ for 1.x
```

Comment:
My mistake, will fix. PS: it would probably take you just as long to make a PR to correct as it did to tell me I was wrong. :wink: